mindsdb/api/mongo/start.py

```diff
@@ -0,0 +1,13 @@
+from mindsdb.utilities.config import Config
+from mindsdb.api.mongo.server import run_server
+
+
+def start(config, initial=False):
+    if not initial:
+        print('\n\nWarning, this process should not have been started... nothing is "wrong" but it needlessly ate away a tiny bit of precious comute !\n\n')
```

Comment:
Does this ever happen ? Also `compute` not `comute`.

Reply:
typo fixed